server.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const browserSync = require('browser-sync');
+
+browserSync({
+  server: 'src',
+  files: ['src/*.html', 'src/*.css'],
```

Comment:
we should have it as external config in node modules

Reply:
ok